Observer/Order/AfterSubmitObserver.php

```diff
@@ -0,0 +1,90 @@
+<?php
+declare(strict_types=1);
```

Comment:
As defined in the [PSR-12](https://www.php-fig.org/psr/psr-12/#3-declare-statements-namespace-and-import-statements)/PER-CS coding standards, there should be a space between the opening PHP tag and the `declare` statement.